docs/release/trg-9/trg-9-01.md

```diff
@@ -0,0 +1,75 @@
+---
+title: TRG 9.01 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+| Status | Created       | Post-History                                             |
+|--------|---------------|----------------------------------------------------------|
+| Active | 22-Jul-2024   | Finalize TRG                                             |
+| Draft  | 10-Jan-2024   | Transferred from Release Acceptance Criteria into TRG    |
+| Draft  | 25-April-2024 | Added new styleguide criteria - initial version          |
+
+## Introduction
+
+The purpose of this guideline is to ensure that business applications with user interfaces adhere to the established UI/UX styleguide. It outlines all the relevant criteria that must be checked when releasing a new product version. A crosscheck with every pull request is not just optional, but it is a "MUST" criteria to ensure the quality and consistency of the user interfaces of each product/component. This step is crucial in maintaining the overall standard and enhancing as well as standardizing the user experience.
+
+## Objective
+
+The objective is to ensure consistency, usability, and adherence to design standards across all user interfaces within the business application.
+
+## Criteria (Basic)
+
+### Styleguide Compliance
+
+- All user interface elements must match the predefined styles set in the UI/UX styleguide. This includes fonts, colors, button styles, input fields, and layout grids.
```

Comment:
If I got things correctly these are explained in the storybook of the shared components but this is not really obvious to me. Thus, I would like to see a link in the beginning, that the styleguide can be seen here. If it's the storybook for the UI one should follow it's ok as a starter. Seperating concerns even more, I would prefer to have the TRG containing the Styleguide or it being an own repository for which the shared components are an implementation (like the reference implementation in KITs).

Reply:
thanks for your feedback, I tried to integrate it